### PR TITLE
Dev spring boot readme

### PR DESCRIPTION
**Pull request by OlegTuryk**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+**OrderController**: Handles order operations: create, get, update; handles order item get operations.
+
+**ShoppingCartController**: Handles shopping cart get operation, and cart item operations: create, delete, update.
+
+
+## Prerequisites
+
+- [Java 17](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html)
+- [Maven](https://maven.apache.org/download.cgi)
+- [MySQL database](https://www.mysql.com/downloads/)
```
Comment (vovasalyha) [inline readme.md]: not required if running with docker

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+**OrderController**: Handles order operations: create, get, update; handles order item get operations.
+
+**ShoppingCartController**: Handles shopping cart get operation, and cart item operations: create, delete, update.
+
+
+## Prerequisites
+
+- [Java 17](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html)
+- [Maven](https://maven.apache.org/download.cgi)
```
Comment (vovasalyha) [inline readme.md]: not required if using maven wrapper

**Review by vovasalyha — CHANGES_REQUESTED** [review]
GJ. Where is Loom video?Also, if possible, make your readme more engaging by adding some visuals etc.

**Review by andrii-hoienko — APPROVED** [review]
GJ!

**Pull request merged**